### DISCUSSION TITLE
Search not loading results (running through IIS)

### DISCUSSION THREAD

**Discussion by sophiepd13** [Pagefind/pagefind#741] [Q&A]
I built a site using Astro and Pagefind, and when running a local build through VS code the search worked fine, but since hosting the site through IIS, the search just gets stuck on 'Searching for X ....". Anyone had a similar problem and found a solution? Thanks!

**Comment by bglw** [comment]
Hello 👋 

Yes, I have been meaning to write up some specific documentation on IIS. The issue is that it blocks assets for which it doesn't know the MIME type.

You can see some discussion resolving this here: https://github.com/CloudCannon/pagefind/issues/527

And here: https://stackoverflow.com/questions/21266415/asp-net-404-when-downloading-file-with-unknown-extension/21266780#21266780

**Reply by sophiepd13** [reply]
Following these has resolved my issues! I just needed to add the MIME application types in IIS. Thank you so much for the quick solution!
